packages/pipeline-editor/src/CustomFormControls/NumberControl.tsx

```diff
@@ -49,8 +49,9 @@ const ErrorMessage = styled.div`
   z-index: 1;
   border-style: solid;
   border-width: 1px;
-  border-color: #be1100;
-  background-color: #5a1d1d;
+  border-color: ${({ theme }) => theme.palette.errorBorder};
+  background-color: ${({ theme }) => theme.palette.error.main};
+  color: ${({ theme }) => theme.palette.error.contrastText};
```

Comment:
Can we turn this into a shared component so all error messages look the same?

also, I think the colors we are using are meant for something else. I need to double check, but we might want to rename the `error` group to something else and create a new group for these messages

Reply:
I decided to leave `error` group name as is, then added an `errorMessage` group. What do you think?
Also moved `ErrorMessage` to `validators` package - I wasn't sure what would be the best place for it, let me know if it belongs better somewhere else.